pkg/template/static_context.go

```diff
@@ -110,6 +110,8 @@ func (ctx StaticCtx) FuncMap() template.FuncMap {
 
 	funcMap["Lookup"] = ctx.lookup
 
+	funcMap["AdditionalTLSCACert"] = ctx.additionalTLSCACert
```

Comment:
let's match the flag so it's more intuitive to link this all together (not a suggestion you would commit since I'm just changing this in one place)
```suggestion
	funcMap["PrivateCACert"] = ctx.additionalTLSCACert
```